dotAPNS/ApplePush.cs

```diff
@@ -387,7 +387,9 @@ public object GeneratePayload()
                     apsAsDict[customApsProperty.Key] = customApsProperty.Value;
             }
 
-            return payload;
+            var dict = (IDictionary<string, object>)payload;
```

Comment:
Wouldn't that remove the `aps` section for all push types instead of only mdm?

Reply:
You could add something like :

```
var dict = (IDictionary<string, object>)payload;
if (Type == ApplePushType.Mdm)
    dict.Remove("aps");
 return dict;
```